template/script.sh.erb

```diff
@@ -5,19 +5,13 @@ modules_to_load, alternate_r_version = context.version.split(':')
 # Load the required environment
 setup_env () {
   module purge
-  # If you want to use Singularity it is recommended you setup a module similar
-  # to the included `example_module` and load it as:
-  #
-  #     module use "/path/to/example_module/modulefiles"
-  #     module load singularity rstudio_singularity/3.4.3
-  #
-  # Bonus points if you install the `example_module` on the system so you don't
-  # need to run `module use`.
   module load <%= modules_to_load %>
-  <% if alternate_r_version %>
+<% if alternate_r_version %>
   module switch <%= alternate_r_version %>
   export RSTUDIO_WHICH_R=/usr/local/<%= alternate_r_version %>
-  <% end %>
+<% end %>
+
+  module load singularity
```

Comment:
No longer needed.  If you do `which singularity` you will see it's now in `/usr/bin`.  Maybe in future some useful environment variables would be set but at this time the module isn't necessary.